src/cf-typography/src/atoms/micro-copy.less

```diff
@@ -0,0 +1,8 @@
+.a-micro-copy {
+    color: @micro-copy;
+    font-size: unit( @size-v / @base-font-size-px, em );
+
+    &__large {
+        font-size: unit( @base-font-size-px / @base-font-size-px, em );
+    }
```

Comment:
It's pre-existing and in use in a few places https://github.com/cfpb/cfgov-refresh/search?utf8=%E2%9C%93&q=micro-copy__large&type=Code